### DISCUSSION TITLE
Brightness keys not working after MonitorControl unistall

### DISCUSSION THREAD

**Discussion by doud62** [MonitorControl/MonitorControl#1240] [Q&A]
Hi,

I tried MonitorControl on my Mac mini M1 but I unistalled it because I don't need it anymore. Unfortunately my brightness keys aren't working anymore now... I use the Apple Magic Keyboard and those keys worked before I installed MonitorControl 😕 

Maybe MonitorControl change something in the settings and now it's broken 😕 

Do you know how I can fix this?

Thanks!

**Comment by waydabber** [comment]
What kind of display are you using?

**Reply by doud62** [reply]
This one: https://www.samsung.com/au/business/monitors/ultra-wide/s9-s95ua-ls49a950uiexxy/
